src/lib/ui/ClimateSettingForm/ClimateSettingScheduleFormDeviceSelect.tsx

```diff
@@ -0,0 +1,38 @@
+import { type Control, Controller, type FieldValues } from 'react-hook-form'
+
+import { ContentHeader } from 'lib/ui/layout/ContentHeader.js'
+import { ThermostatSelect } from 'lib/ui/thermostat/ThermostatSelect.js'
+
+interface ClimateSettingScheduleFormDeviceSelectProps {
+  title: string
+  control: Control<FieldValues, any>
```

Comment:
I think the second param should be optional unless we are using the context.
```suggestion
  control: Control<FieldValues>
```